File: GFG/Max Min/Max Min.java

class Solution
{ 
    public static int findSum(int A[],int N) 
    {
        //code here
        int min=Integer.MAX_VALUE;
        int max=Integer.MIN_VALUE;
        for(int x:A){
            min=Math.min(min,x);
            max=Math.max(max,x);
        }
        return min+max;
    }
}

File: GFG/Max Min/Max Min.py
class Solution:
    def findSum(self,A,N): 
        #code here
        min = A[0]
        max = A[0]
        for x in range(1,N):
            if(min>A[x]):min=A[x]
            if(max<A[x]):max=A[x]
        return max+min;